connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConfig.java

```diff
@@ -400,6 +412,53 @@ public WorkerConfig(ConfigDef definition, Map<String, String> props) {
         logInternalConverterDeprecationWarnings(props);
     }
 
+    @Override
+    public String toString() {
+        return "Comma-separated header rules, where each header rule is of the form "
+                + "'[action] [header name]:[header value]' and optionally surrounded by double quotes "
+                + "if any part of a header rule contains a comma";
+    }
+
+    public static void validateHttpResponseHeaderConfig(String config) {
```

Comment:
This could be package-level protected, right? The only place it should be called is in `ResponseHttpHeadersValidator` and in tests.
```suggestion
    // Visible for testing
    static void validateHttpResponseHeaderConfig(String config) {
```

Reply:
Cannot pass compile after changed to either default package level or protected due to RestServerTest call it and they are on different package.

Reply:
Then how about making these package-level protected (as I mentioned) and moving the tests of these methods to `WorkerConfigTest` instead?

Reply:
Actually should move these header config verification testing to WorkerConfigTest.